docs/guide/known-limitations.md

```diff
@@ -72,11 +72,7 @@ To remove the differences, you can create custom implementations of those functi
 | DAYS          | =DAYS(-1, 0)                                               |          NUM |      -1,0000 |        NUM |
```

Comment:
The pages https://handsontable.github.io/hyperformula/guide/specifications-and-limits.html and https://handsontable.github.io/hyperformula/guide/known-limitations.html appear disturbingly similar in topic. I know that one of them is about *limits* and the other about *limitations*, but I guess that the reader is interested in both. Maybe the pages should be cross linked at least?

Anyway, should something about `RATE` be added to the other page (https://handsontable.github.io/hyperformula/guide/specifications-and-limits.html)?

Reply:
> Maybe the pages should be cross linked at least?

Issue: Maybe the pages should be cross linked at least?

> Anyway, should something about RATE be added to the other page

Well, IMO the values of `epsMax` and `iterMax` parameters should appear in our documentation. But I would rather put them in the section about `RATE` function (which doesn't exist). @kirszenbaum what do you think?

Reply:
### `RATE`

For now, we don't have any sections devoted to individual functions. The only places where we describe any details of individual functions are:
- [Bulit-in functions](https://handsontable.github.io/hyperformula/guide/built-in-functions.html)
- [Known limitations](https://handsontable.github.io/hyperformula/guide/known-limitations.html#nuances-of-the-implemented-functions)

IMO, if `RATE` works differently than GS or Excel (`iterMax`), we should add an entry about it to this table:
- [Google Sheets and Microsoft Excel](https://handsontable.github.io/hyperformula/guide/known-limitations.html#google-sheets-and-microsoft-excel)

Any other information about `RATE` – I'd add it here:
- [Nuances of the implemented functions](https://handsontable.github.io/hyperformula/guide/known-limitations.html#nuances-of-the-implemented-functions)

### Known limitations

I'd definitely change the title of the [Known limitations](https://handsontable.github.io/hyperformula/guide/known-limitations.html) page:
- It looks bad (especially in the **Getting started** section)
- It doesn't match the contents of the page (implementation differences and "function nuances" are not limitations)
- It indeed sounds similar to "Specifications and limits"

My proposition is to:
- Rename "Known limitations" to "Implementation details" or similar
- Move this page from the **Getting started** section to the **Advanced topics** section
- Change the introduction sentence to actually cover what's on the page (also, there's a typo in the 1st sentence at the moment: "This pages lists")
- Turn the top bullet list ("Node.js versions older than...") into a separate sub-section of the page (and **that** section could be called "Known limitations", or "Areas for improvement", or "Areas for further development")

Reply:
I described the implementation details in 'Nuances of the implemented functions':

>* RATE function might have no solutions (depending on the inputs). Sometimes it has multiple solutions. Our implementation uses an iterative algorithm (Newton's Method) for finding an approximation for one of them to within `1e-7`. If it is not found after `50` iterations function returns the `#NUM!` error.

But - really - from the perspective of our user, it would make much more sense to keep all the function-related information in the description of a function instead of spreading them across [List of available functions](https://handsontable.github.io/hyperformula/guide/built-in-functions.html#list-of-available-functions). [Nuances of the implemented functions](https://handsontable.github.io/hyperformula/guide/known-limitations.html#nuances-of-the-implemented-functions) and [Google Sheets and Microsoft Excel](https://handsontable.github.io/hyperformula/guide/known-limitations.html#google-sheets-and-microsoft-excel) sections.

Reply:
Thank you for bright ideas about these pages. It is definitely off-topic though so let's track it in the new issue.